### PR TITLE
Upadate backend BUILD files

### PR DESCRIPTION
**Pull request by chensun**
updated/generated by running `bazel run //:gazelle -- update-repos --from_file=go.mod`

/cc @rui5i @Ark-kun

### CODE REVIEW

File: backend/src/cache/server/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    ],
+    importpath = "github.com/kubeflow/pipelines/backend/src/cache/server",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//backend/src/cache/client:go_default_library",
+        "//backend/src/cache/model:go_default_library",
+        "//backend/src/cache/storage:go_default_library",
+        "//backend/src/common/util:go_default_library",
+        "@com_github_golang_glog//:go_default_library",
+        "@com_github_peterhellberg_duration//:go_default_library",
```
Comment (rmgogogo) [inline backend/src/cache/server/BUILD.bazel:19]: wondering whether we can change codes to use golang prebuilt Duration to avoid introducing 3rd lib.

Comment (rmgogogo) [reply backend/src/cache/server/BUILD.bazel:19]: https://golang.org/pkg/time/#Duration.Hours

Comment (rmgogogo) [reply backend/src/cache/server/BUILD.bazel:19]: https://stackoverflow.com/questions/28125963/golang-parse-time-duration

Comment (rmgogogo) [reply backend/src/cache/server/BUILD.bazel:19]: never mind, that 3P lib is simple and small.

**Comment by rmgogogo** [comment]
/lgtm

Jing is BE owner. may ask Jing for an approval

File: backend/src/cache/server/BUILD.bazel (hunk 1 above)
Comment (rui5i) [reply backend/src/cache/server/BUILD.bazel:19]: Hi Renmin, this 3P is used for parse ISO8601 duration(eg. P30DT1H) to golang time.Duration which is not built in golang's Duration pkg.

**Comment by rui5i** [comment]
Thanks @chensun !

**Comment by Ark-kun** [comment]
Thank you!

**Pull request merged**